crates/client/rpc/src/lib.rs

```diff
@@ -98,6 +101,19 @@ where
     pub fn current_block_number(&self) -> RpcResult<u64> {
         Ok(UniqueSaturatedInto::<u64>::unique_saturated_into(self.client.info().best_number))
     }
+
+    fn load_genesis() -> GenesisData {
+        let genesis_path: PathBuf = String::from(GENESIS_FILE).into();
+        let genesis_file_content = std::fs::read_to_string(genesis_path.clone()).unwrap_or_else(|_| {
+            panic!(
+                "Failed to read genesis file at {}. Please run `madara setup` before opening an issue.",
+                genesis_path.canonicalize().unwrap().display()
+            )
+        });
+        let genesis_data: GenesisData = serde_json::from_str(&genesis_file_content).expect("Failed loading genesis");
+
+        genesis_data
+    }
```

Comment:
I guess this is the same function used inside the node crate. As @tdelabro mentioned, let's reuse that.

Reply:
And it use a dynamic path rather than the one you hardcoded

Reply:
Ya we should do that imo. I think we can have a utils crate at the same level as `node`, `client` etc. for functions that are used across crates. I am not sure if this is the idiomatic approach, @tdelabro?

Reply:
Awesome, @Juul-Mc-Goa can you make this change?

Reply:
yep working on it